runelite-client/src/main/java/net/runelite/client/plugins/hiscore/HiscorePanel.java

```diff
@@ -100,13 +93,10 @@
 @Slf4j
 public class HiscorePanel extends PluginPanel
 {
-	private static final String SKILL_NAME = "SKILL_NAME";
-	private static final String SKILL = "SKILL";
-
 	/**
 	 * Real skills, ordered in the way they should be displayed in the panel.
 	 */
-	private static final Set<HiscoreSkill> SKILLS = new LinkedHashSet<>(Arrays.asList(
+	private static final ArrayList<HiscoreSkill> SKILLS = new ArrayList<>(Arrays.asList(
```

Comment:
Any reason why the type is an ArrayList instead of a List? It should be kept as the interface. It should look like: `private static final List<HiscoreSkill> SKILLS = new ArrayList<>(Arrays.asList...`